src/pam-ssh-add/pam-ssh-add.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef PAM_SSH_ADD_H__
+#define PAM_SSH_ADD_H__
+
+#include "pwd.h"
+
+extern const char *pam_ssh_agent_program;
+extern const char *pam_ssh_agent_arg;
+extern const char *pam_ssh_add_program;
+extern const char *pam_ssh_add_arg;
+
+
+void    pam_ssh_add_logger          (void (* func) (int level, const char *data),
+                                     int verbose);
```

Comment:
It's a small thing, but I would suggest just exposing more variables instead of having this function. That makes it really clear what it's used for ... testing.